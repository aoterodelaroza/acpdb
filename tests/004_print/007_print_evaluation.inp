## check: 007_print_evaluation.out -a1e-10
## labels: regression quick

connect ../zz_source/bsip2-small/bsip2.db
print evaluation
