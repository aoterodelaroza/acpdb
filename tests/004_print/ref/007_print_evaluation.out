* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

* PRINT: print the contents of the database 

| methodid| propid| #values| 1st value|
| 1| 1| 1| -3.133|
| 1| 2| 1| -4.989|
| 1| 3| 1| -18.753|
| 1| 4| 1| -16.062|
| 1| 5| 1| -20.641|
| 1| 6| 1| -16.934|
| 1| 7| 1| -16.66|
| 1| 8| 1| -0.527|
| 1| 9| 1| -1.472|
| 1| 10| 1| -1.448|
| 1| 11| 1| -2.654|
| 1| 12| 1| -4.255|
| 1| 13| 1| -9.805|
| 1| 14| 1| -4.524|
| 1| 15| 1| -11.73|
| 1| 16| 1| -1.496|
| 1| 17| 1| -3.275|
| 1| 18| 1| -2.312|
| 1| 19| 1| -4.541|
| 1| 20| 1| -2.717|
| 1| 21| 1| -5.627|
| 1| 22| 1| -7.097|
| 1| 23| 3| 0.1|

